extract-method-plugin/src/main/kotlin/org/jetbrains/research/extractMethod/BaseRunner.kt

```diff
@@ -0,0 +1,46 @@
+package org.jetbrains.research.extractMethod
+
+import org.apache.logging.log4j.LogManager
+import org.jetbrains.research.extractMethod.core.extractors.NegativeRefactoringsExtractionRunner
+import org.jetbrains.research.extractMethod.core.extractors.PositiveRefactoringsExtractionRunner
+import org.jetbrains.research.pluginUtilities.openRepository.getKotlinJavaRepositoryOpener
+import org.jetbrains.research.pluginUtilities.preprocessing.getKotlinJavaPreprocessorManager
+import java.io.FileWriter
+import java.nio.file.Files
+import java.nio.file.Path
+import kotlin.streams.toList
+
+private val preprocessor = getKotlinJavaPreprocessorManager(null)
+private val repositoryOpener = getKotlinJavaRepositoryOpener()
+
+class BaseRunner {

```

Comment:
maybe we should name it as `RepositoryOpener`?
also, these two methods look similar. we can just leave only one and add a new parameter `extractor: RefactoringExtractor` (see the comment below about common interface for refactoring extractors)